code/modules/reagents/Chemistry-Recipes_vr.dm

```diff
@@ -205,7 +205,14 @@
 				O.show_message(text("<span class='warning'>The slime core fizzles disappointingly.</span>"), 1)
 			return
 
-		var/blocked = list(/obj/item/stack/material, /obj/item/stack/material/cyborg, /obj/item/stack/material/cyborg/plastic, /obj/item/stack/material/cyborg/plasteel, /obj/item/stack/material/cyborg/glass/reinforced, /obj/item/stack/material/cyborg/wood, /obj/item/stack/material/animalhide/human, /obj/item/stack/material/animalhide/corgi, /obj/item/stack/material/animalhide/cat, /obj/item/stack/material/animalhide/monkey, /obj/item/stack/material/animalhide/lizard , /obj/item/stack/material/animalhide/xeno, /obj/item/stack/material/cyborg, /obj/item/stack/material/cyborg/glass/reinforced)
+		var/blocked = list(
+							/obj/item/stack/material,					//Technical stacks
+							/obj/item/stack/material/animalhide,
+							/obj/item/stack/material/morphium,			//Complex materials requiring Particle Smasher to create
+							/obj/item/stack/material/morphium/hull,
+							/obj/item/stack/material/valhollide,
+							/obj/item/stack/material/supermatter)
```

Comment:
Staff had a discussion and it was decided that particle smasher materials should be kept as obtainable via xenobio.

However, an acceptable alternative would be to set it up so that particle smasher materials outputs a lower amount of material, instead of 1-50 sheets per reaction, perhaps 1-15 sheets instead.